galaxy/wrapper/hicFindTADs.xml

```diff
@@ -76,39 +90,48 @@
         <data name="outFileName" from_work_dir="tadScoreFile.tabular" format="tabular">
             <filter>mode['mode_selector'] == "TAD_score"</filter>
         </data>
-        <data name="boundaries" from_work_dir="galaxy_tad_prefix_boundaries.bed" format="bed" label="${tool.name} on ${on_string}: Boundary positions">
+        <data name="tad_score_zscore_matrix" from_work_dir="tadScoreFile.tabular_zscore_matrix.h5" format="h5">
+            <filter>mode['mode_selector'] == "TAD_score"</filter>
+        </data>
+        <data name="boundaries" from_work_dir="galaxy_tad_prefix_boundaries.bed" format="bed"
+            label="${tool.name} on ${on_string}: Boundary positions">
             <filter>mode['mode_selector'] == "find_TADs"</filter>
         </data>
-        <data name="score" from_work_dir="galaxy_tad_prefix_score.bedgraph" format="bedgraph" label="${tool.name} on ${on_string}: Matrix with multi-scale TAD scores">
+        <data name="score" from_work_dir="galaxy_tad_prefix_score.bedgraph" format="bedgraph"
+            label="${tool.name} on ${on_string}: Matrix with multi-scale TAD scores">
             <filter>mode['mode_selector'] == "find_TADs"</filter>
         </data>
-        <data name="domains" from_work_dir="galaxy_tad_prefix_domains.bed" format="bed" label="${tool.name} on ${on_string}: TAD domains">
+        <data name="domains" from_work_dir="galaxy_tad_prefix_domains.bed" format="bed"
+            label="${tool.name} on ${on_string}: TAD domains">
             <filter>mode['mode_selector'] == "find_TADs"</filter>
         </data>
-        <data name="boundaries_bin" from_work_dir="galaxy_tad_prefix_boundaries_bin.bed" format="bed" label="${tool.name} on ${on_string}: Linkage information">
+        <data name="boundaries_bin" from_work_dir="galaxy_tad_prefix_boundaries.gff"
+            format="bed" label="${tool.name} on ${on_string}: Boundary information plus score">
             <filter>mode['mode_selector'] == "find_TADs"</filter>
         </data>
     </outputs>
     <tests>
-        <test>
+        <!--test>
             <param name="matrix" value="hicBuildMatrix_result1.h5" ftype="h5"/>
             <param name="mode_selector" value="TAD_score"/>
             <param name="minDepth" value="20000"/>
             <param name="maxDepth" value="60000"/>
-            <param name="step" value="10000"/>
+            <param name="step" value="100000"/>
             <output name="outFileName" file="hicFindTADs_TAD_score.tabular" ftype="tabular"/>
+            <output name="tad_score_zscore_matrix" file="tadScoreFile.tabular_zscore_matrix.h5" ftype="h5" compare="sim_size"/>
         </test>
         <test>
             <param name="tadScoreFile" value="hicFindTADs_TAD_score.tabular" ftype="tabular"/>
+            <param name="tad_score_zscore_matrix" value="tadScoreFile.tabular_zscore_matrix.h5" ftype="h5"/>
```

Comment:
is that file generated as part of the testing? if a low resolution matrix is used, a smaller zscore matrix for hicFindTads can be created.